www/build-themes.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
+const sass = require('node-sass');
```

Comment:
FYI, now libsass was replaced with dart sass, `node-sass` can be changed to `sass` here.

Reply:
True, updated it, thanks!